### ISSUE TITLE
[osgearth:x64-linux] Build failure in linux 64 bit: does not find osg version

### ISSUE THREAD

**Issue by omaralvarez** [microsoft/vcpkg#11614]
**Host Environment**
- OS: Ubuntu 18.04
- Compiler: gcc 7.5.0

**To Reproduce**
`./vcpkg install osgearth`

**Failure logs** 
```
Building package osgearth[core]:x64-linux...
-- Note: osgearth only supports dynamic library linkage. Building dynamic library.
CMake Error at ports/osgearth/portfile.cmake:8 (message):
  Could not determine osg version
Call Stack (most recent call first):
  scripts/ports.cmake:90 (include)


Error: Building package osgearth:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: osgearth:x64-linux
  Vcpkg version: 2020.02.04-unknownhash

Additionally, attach any relevant sections from the log files above.
CMake Error at build/_pmm/1.9.4/vcpkg.cmake:232 (message):
  Failed to install requirements with vcpkg [1]:

Call Stack (most recent call first):
  build/_pmm/1.9.4/main.cmake:41 (_pmm_vcpkg)
  build/_pmm/1.9.4/main.cmake:57 (_pmm_project_fn)
  CMakeLists.txt:11 (PMM)


-- Configuring incomplete, errors occurred!
See also "/home/.../cpp/gpsop/build/CMakeFiles/CMakeOutput.log".
```

**Additional context**
Fixed copying `lib/osgPlugins-*` to `tools/osg` since it does not check the lib folder which I think should be better, should I make a pull request?

**Comment by PhoebeHui** [comment]
@omaralvarez, thanks for reporting this issue!

Could you please make a PR for this?

**Comment by jepessen** [comment]
What should we do in order to make the pull request? If you can't do it I can do, because I need this PR.

**Comment by omaralvarez** [comment]
Done!

**Comment by jepessen** [comment]
ok thanks

**Comment by nduboys** [comment]
I have copied the directory with all libs from lib/osgPlugins-3.6.5 to tools/osg.
So I have a installed/x64-linux/tools/osg/osgPlugins-3.6.5 directory with all ibs inside.
But the build fails again with a lot of unresolved symbols
Is there any thing else to do please ?

**Comment by joe-ieta** [comment]
The same problem.  and I copied the directory as mentioned above, the build fails :

...
Building package osgearth[core]:x64-linux...
-- Note: osgearth only supports dynamic library linkage. Building dynamic library.
-- Using cached /home/vcpkg/downloads/gwaldron-osgearth-342fcadf4c8892ba84841cb5b4162bdc51519e3c.tar.gz
-- Cleaning sources at /home/vcpkg/buildtrees/osgearth/src/dc51519e3c-7297c5bc76.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/vcpkg/downloads/gwaldron-osgearth-342fcadf4c8892ba84841cb5b4162bdc51519e3c.tar.gz
-- Applying patch make-all-find-packages-required.patch
-- Applying patch fix-tinyxml.patch
-- Using source at /home/vcpkg/buildtrees/osgearth/src/dc51519e3c-7297c5bc76.clean
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
_**CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:144 (message):
    Command failed: /home/vcpkg/downloads/tools/cmake-3.19.2-linux/cmake-3.19.2-Linux-x86_64/bin/cmake --build . --config Debug --target install -- -v -j9**_
    Working Directory: home/vcpkg/buildtrees/osgearth/x64-linux-dbg
    See logs for more information:
      .../vcpkg/buildtrees/osgearth/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:96 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:27 (vcpkg_build_cmake)
  ports/osgearth/portfile.cmake:37 (vcpkg_install_cmake)
  scripts/ports.cmake:131 (include)
.....

I execute the command in a shell , also fails with a lot of unresolved symbols:


/usr/bin/ld: lib/libosgEarthd.so.3.1.0: undefined reference to `SSL_get_shutdown'

/usr/bin/ld: lib/libosgEarthd.so.3.1.0: undefined reference to `jpeg_destroy_decompress'

_**....
   nearly 5 hundred   lines omit
....**_
/usr/bin/ld: lib/libosgEarthd.so.3.1.0: undefined reference to `X509_get_version'

/usr/bin/ld: lib/libosgEarthd.so.3.1.0: undefined reference to `proj_alter_name'

/usr/bin/ld: lib/libosgEarthd.so.3.1.0: undefined reference to `png_get_valid'

/usr/bin/ld: lib/libosgEarthd.so.3.1.0: undefined reference to `proj_crs_get_geodetic_crs'
**collect2: error: ld returned 1 exit status**
[9/13] /usr/bin/c++ -DOSGEARTH_HAVE_MBTILES -Ibuild_include -I/home/vcpkg/buildtrees/osgearth/src/dc51519e3c-7297c5bc76.clean/src -isystem /home/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -g -std=gnu++11 -MD -MT src/applications/osgearth_3pv/CMakeFiles/application_osgearth_3pv.dir/osgearth_3pv.cpp.o -MF src/applications/osgearth_3pv/CMakeFiles/application_osgearth_3pv.dir/osgearth_3pv.cpp.o.d -o src/applications/osgearth_3pv/CMakeFiles/application_osgearth_3pv.dir/osgearth_3pv.cpp.o -c /home/vcpkg/buildtrees/osgearth/src/dc51519e3c-7297c5bc76.clean/src/applications/osgearth_3pv/osgearth_3pv.cpp
**ninja: build stopped: subcommand failed.**

it seems all the refrences are not found,  any idea ? please !

**Comment by JackBoosY** [comment]
@joe-ieta Can you please provide _vcpkg/buildtrees/osgearth/install-x64-linux-dbg-out.log_?

Thanks.

**Comment by PhoebeHui** [comment]
This issue should be fixed by PR#27156. Currently it could be successfully installed with latest vcpkg.
